sourcegraph/conn_pool.go

```diff
@@ -0,0 +1,43 @@
+package sourcegraph
+
+import (
+	"sync"
+
+	"google.golang.org/grpc"
+)
+
+var (
+	connsMu sync.Mutex
+	conns   map[string]*grpc.ClientConn
```

Comment:
Can you also add a comment what the key represents, since `string` type is not self-documenting?